src/main/java/com/softserve/skillscope/talentInfo/model/entity/TalentInfo.java

```diff
@@ -1,20 +1,49 @@
 package com.softserve.skillscope.talentInfo.model.entity;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import com.softserve.skillscope.talent.model.entity.Talent;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import java.time.LocalDate;
 
 @Setter
 @Getter
+@Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @Entity
 public class TalentInfo {
     @Id
     @GeneratedValue
-    private Integer id;
+    private Long id;
+
+    @MapsId
+    @OneToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "talent_id", nullable = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    private Talent talent;
+
+    private String image;
```

Comment:
Add @NotEmpty annotation to this field due to using the static image in our db.

Reply:
Done